src/pages/dho/Members.vue

```diff
@@ -15,35 +17,29 @@ export default {
     daoMembers: {
       query: require('../../query/members/members-pagination.gql'),
       update: data => {
-        return data.getDao
-      },
-      variables () {
```

Comment:
why do you remove the variables?? how do you filtering the members for the selected dao??